examples/java/java/App.java

```diff
@@ -0,0 +1,86 @@
+
+
+import java.io.IOException;
+import software.amazon.awssdk.core.sync.RequestBody;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.s3.model.CreateBucketConfiguration;
+import software.amazon.awssdk.services.s3.model.CreateBucketRequest;
+import software.amazon.awssdk.services.s3.model.DeleteBucketRequest;
+import software.amazon.awssdk.services.s3.model.DeleteObjectRequest;
+import software.amazon.awssdk.services.s3.model.HeadBucketRequest;
+import software.amazon.awssdk.services.s3.model.PutObjectRequest;
+import software.amazon.awssdk.services.s3.model.S3Exception;
+import software.amazon.awssdk.services.s3.S3Client;
+
+
+public class App {
+
+  public static void main(String[] args) throws IOException {
+
+    Region region = Region.US_WEST_2;
+    S3Client s3 = S3Client.builder().region(region).build();
+
+    String bucket = "bucket" + System.currentTimeMillis();
+    String key = "key";
```

Comment:
i guess that "key" is the object name, where do you specify the content of the object?